docs/config_reference.rst

```diff
@@ -1179,6 +1179,15 @@ The additional properties for the ``httpjson`` handler are the following:
    A set of optional key/value pairs to be passed with each log record to the server.
    These may depend on the server configuration.
 
+.. js:attribute:: .logging[].handlers[].perflog_ignore
```

Comment:
Since it's already a perflog-only property, I would simply name it as `ignore_keys`.